Procfile.dev

```diff
@@ -1,3 +1,3 @@
 web: python3 manage.py runserver 0.0.0.0:5000
-worker: python3 manage.py run_huey --no-periodic
+worker: python3 manage.py run_huey --no-periodic --workers=6
```

Comment:
@Kelketek, same as [this comment](https://github.com/open-craft/opencraft/pull/628/files#r469852862). In fact, this may not be needed in a dev environment at all.

Reply:
> > This pull request enables multiple threads per worker.
> 
> @fox, can we update the log message format to include the thread id if it is available? With that it is easier to identify and filter log messages by thread.

```
13:25:04 worker.1              | [2020-08-13 13:25:04,569] INFO:instance.models.load_balancer:Worker-5:load_balancer=1 (haproxy-dev.ope) | PLAY [all] *****************************************************************************************************************************
```

While Honcho considers this worker 1, Huey considers it as Worker-5 (as this task was executed in the fifth thread), and marks that in the logs.

> @Kelketek, same as this comment. In fact, this may not be needed in a dev environment at all.

Perhaps not, after this change, but it'd probably be good to have more than one, if someone wants to test multi-appserver deployment locally, otherwise they'll be stuck with serial provisioning. While local deployment may not be working yet, that is a target of my other task.

> @Kelketek, instead of hardcoding this, can we make this configurable?

I should be able to find a way to do this. Will update once I've got it. :)